app/src/main/java/com/brainwallet/ui/theme/Theme.kt

```diff
@@ -254,10 +254,10 @@ val unspecified_scheme = ColorFamily(
 
 @Composable
 fun BrainwalletAppTheme(
-    darkTheme: Boolean = true,
+    darkTheme: Boolean = true, //TODO: will change this later, will be using toggle
 //    darkTheme: Boolean = isSystemInDarkTheme(),
     // Dynamic color is available on Android 12+
-    dynamicColor: Boolean = true,
+    dynamicColor: Boolean = true, //dynamic color will disable or not?
```

Comment:
probably will remove this, since we are not supporting dynamic color on android 12+
just using light/dark mode

Reply:
Yes...baby steps. Just dark and light mode @andhikayuana